CONTRIBUTING.md

```diff
@@ -81,7 +81,11 @@ private EntityUid Slice(...)
 
 If you want to make changes to a map, get in touch with its maintainer to make sure you don't both make changes at the same time.
 
-Conflicts with maps make PRs mutually exclusive so either your work on the maintainer's work will be lost, communicate to avoid this!
+Conflicts with maps make PRs mutually exclusive so either your work or the maintainer's work will be lost, communicate to avoid this!
+
+Please make a detailed list of **all** changes(even minor changes) with locations when submitting a PR. This helps reviewers hone in on them withot having to search an entire map for differences. Ex: [Map Edits](https://github.com/DeltaV-Station/Delta-v/pull/3165)
+
+Please limit changelogs on map PRs to **significant** map alterations or additions. Minor map edits do not need changelogs. Limit changelogs to one per PR.
```

Comment:
is this with the upcoming MAPS: changelog in mind

Reply:
Yes/no. I've been meaning to add this for a while. But the addition will be good